flink-table/flink-table-planner/src/main/scala/org/apache/flink/table/plan/rules/datastream/DataStreamWindowJoinRule.scala

```diff
@@ -80,7 +80,7 @@ class DataStreamWindowJoinRule
         leftRowSchema.arity,
         join.getRowType,
         join.getCluster.getRexBuilder,
-        TableConfig.DEFAULT)
```

Comment:
I think we still can have `TableConfig.DEFAULT` instance, can't we?

Reply:
The `DEFAULT` in scala is actually a method. We need to create each time because the tableConfig could be modified in anyplace. 
To make it more convenient, maybe we can add a static `getDefault()` method in `TableConfig`. What do you think?